apps/dashboard/src/contract-ui/tabs/events/components/events-feed.tsx

```diff
@@ -258,14 +257,7 @@ const EventsFeedItem: React.FC<EventsFeedItemProps> = ({
                   bg="transparent"
                   onClick={() => {
                     onCopy();
-                    toast({
-                      variant: "solid",
-                      position: "bottom",
-                      title: "Transaction hash copied.",
-                      status: "success",
-                      duration: 5000,
-                      isClosable: true,
-                    });
+                    toast.success("Transaction hash copied.");
```

Comment:
This shouldn't be "success" variant - if there is other visual feedback for copy action - we can completely drop this - otherwise use "info" variant